client/app/page.tsx

```diff
@@ -1,24 +1,22 @@
-import { Button } from '@/_components/ui'
+import { Button, Heading } from '@/_components/ui'
 
 export default function Home() {
   return (
-    <main className='p-24'>
-      <p className='text-xs p-4'>Hello World!</p>
-      <div className='text-sm m-4 flex gap-10 p-4'>
-        <Button variant={'destructive'}>Click me</Button>
-        <Button variant={'outline'} disabled>
-          Click me
-        </Button>
-        <Button variant={'secondary'} disabled>
-          Click me
-        </Button>
-        <Button variant={'ghost'}>Click me</Button>
-        <Button variant={'link'}>Click me</Button>
-        <Button disabled>Click me</Button>
-        <Button type='submit' isLoading>
-          Click me
-        </Button>
-      </div>
-    </main>
+    <div>
+      <Heading
+        title='Fairy share'
+        // TODO: There might be a better way to switch between two different size of Button component depending on the screen size?
+        buttonComponent={() => (
+          <>
+            <Button variant={'outline'} className='block md:hidden'>
+              Rules
+            </Button>
+            <Button variant={'outline'} className='hidden md:block'>
+              Rules
+            </Button>
+          </>
```

Comment:
I actually don't know why you need the two buttons.  We need show the button both desktop and smartphone, right??

```
            <Button
              variant={'outline'}
              size={'sm'}
              className='rounded-md px-5 md:h-11 md:text-lg'
            >
              Rules
            </Button>
```

Reply:
I fixed it this way!
https://github.com/HAK2024/fairy-share/pull/55/commits/160cbf1849f279bfd78810aebcf0a22d6f3435bb